docs/source/llms/openai/autologging.rst

```diff
@@ -0,0 +1,184 @@
+Enabling autologging with no argument overrides will behave as the ``default`` configuration in the table in the next section. Overriding any of these settings 
+will allow you to log additional elements. 
+
+.. tip::
+    The only element that is **enabled by default** when autologging is activated is the recording of trace information. You can read more about MLflow tracing 
+    `here <../tracing/index.html>`_. 
+
+.. code-block:: python
+
+    import os
+    import openai
+    import mlflow
+
+    mlflow.openai.autolog()
+
+    openai_client = openai.OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
+
+    # Use the openai_client and traces will be logged to the active experiment
+    ...
+
+Configuration of OpenAI Autologging
+-----------------------------------
+
+MLflow OpenAI autologging can log various information about the model and its inference. **By default, only trace logging is enabled**, but you can enable 
+autologging of other information by setting the corresponding parameters when calling :py:func:`mlflow.openai.autolog()`. 
+
+The available options and their default values are shown below. To learn more about additional parameters, see the API documentation.
+
+.. list-table::
+    :widths: 20 20 30 30
+    :header-rows: 1
+
+    * - Target
+      - Default
+      - Parameter
+      - Description
+    * - Traces
+      - ``true``
+      - ``log_traces``
+      - Whether to generate and log traces for the model. See `MLflow Tracing <../tracing/index.html>`_ for more details about the tracing feature.
+    * - Model Artifacts
+      - ``false``
+      - ``log_models``
+      - If set to ``True``, the OpenAI model will be logged when it is invoked. 
+    * - Model Signatures
+      - ``false``
+      - ``log_model_signatures``
+      - If set to ``True``, :py:class:`ModelSignatures <mlflow.models.ModelSignature>` describing model inputs and outputs are collected and logged along with OpenAI model artifacts during inference. This option is only available when ``log_models`` is enabled.
+    * - Input Example
+      - ``false``
+      - ``log_input_examples``
+      - If set to ``True``, input examples from inference data are collected and logged along with OpenAI model artifacts during inference. This option is only available when ``log_models`` is enabled.
+    * - Datasets
+      - ``false``
+      - ``log_datasets``
+      - If set to ``True`` and a dataset is used with interacting with the OpenAI model, information about the dataset will be logged.
+
+
+For example, to disable logging of traces, and instead enable model logging, run the following code:
+
+.. code-block:: python
+
+    import mlflow
+
+    mlflow.openai.autolog(
+        log_traces=False,
+        log_models=True,
+    )
+
+Example of using OpenAI Autologging
+-----------------------------------
+
+.. code-block:: python
+
+    import os
+
+    import mlflow
+    import openai
+
+    OPENAI_MODEL = "gpt-4o"
```

Comment:
This variable is used only once. I'd directly pass `gpt-4o` to `openai_client.chat.completions.create`.